cpu/nrf5x_common/radio/Kconfig.nrf5x

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2021 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config HAVE_NRF5X_RADIO
+    bool
+    select MODULE_NRF5X_RADIO if MODULE_NETDEV_DEFAULT
+    help
+        Indicates that an NRF5X radio is present.
+
+if TEST_KCONFIG
+
+menuconfig MODULE_NRF5X_RADIO
+    bool "NRF5X radio driver"
+    depends on HAVE_NRF5X_RADIO
+    depends on TEST_KCONFIG
+
+if MODULE_NRF5X_RADIO
+
+choice
+    bool "NRF5x radio backend"
+
+config MODULE_NRF802154
+    bool "Implementation of the IEEE 802.15.4 for nRF52 radio"
+    depends on HAS_RADIO_NRF802154
+    depends on HAS_PERIPH_TIMER
+    select MODULE_LUID
+    select MODULE_PERIPH_TIMER
```

Comment:
I think this is missing some netdev ieee802154 dependencies, as it fails to link when enabled. @jia200x may shed some light?

Reply:
It seems simply selecting the ieee 802.15.4 helper module makes the trick: `select MODULE_IEEE802154`. It has no dependencies.